stdlib/public/runtime/CMakeLists.txt

```diff
@@ -62,13 +62,57 @@ set(LLVM_OPTIONAL_SOURCES
     MutexPThread.cpp
     MutexWin32.cpp
     CygwinPort.cpp
+    ImageInspectionELF.cpp
```

Comment:
Isn't `ImageInspectionELF.cpp` included in the core `swift_runtime_sources` above?